cmd/radio.go

```diff
@@ -185,18 +192,55 @@ type erasureConfig struct {
 	clnts  []bucketClient
 }
 
+func clientID(cfg remoteConfig) string {
+	hash := sha256.New()
+	bytes, err := msgpack.Marshal(&cfg)
+	if err != nil {
+		return fmt.Sprintf("%s%s", cfg.Bucket, cfg.Endpoint)
```

Comment:
perhaps this can be written as sha256? rather than a string so it shall always be sha256 which this function returns?